tests/test_ses/test_ses_boto3.py

```diff
@@ -516,6 +516,14 @@ def test_render_template():
     result["RenderedTemplate"].should.contain("<h1>Hello John,</h1>")
     result["RenderedTemplate"].should.contain("Your favorite animal is Lion")
 
+    kwargs = dict(
+        TemplateName="MyTestTemplate", TemplateData=json.dumps({"name": "John"}),
+    )
+
+    with pytest.raises(ClientError) as ex:
+        conn.test_render_template(**kwargs)
+    assert ex.value.response["Error"]["Code"] == "MissingRenderingAttributeException"
```

Comment:
Can we get an assert on the message as well?
Just tested it out against AWS, the message it should give is:
`Attribute 'favoriteanimal' is not present in the rendering data.`

(If there are multiple attributes missing, it will only mention the first one)